### DISCUSSION TITLE
checkbox x-model

### DISCUSSION THREAD

**Discussion by sqllyw** [alpinejs/alpine#1301] [4. General]
Hi,

I got a code like this:

      <input type="checkbox" x-model="thumbnail">

this sets a variable in the x-data, I'd like a method to be called too, what is alphine way of doing this? Thanks

**Comment by KevinBatdorf** [comment]
Maybe like this?

```html
<input type="checkbox" x-model="thumbnail" @change="myMethod($event)">
```
